src/containers/TransactionList.js

```diff
@@ -74,8 +73,8 @@ const TransactionList = ({ url, page, from }) => {
   return (
     <>
       <Card
-        title={`${title(total, "Transaction")} (${offset + 1}-${offset +
-          txs.length})`}
+        title={`${t("transaction", { count: txs.length })} (${offset +
```

Comment:
How does this work exactly? The count piece.

Reply:
https://www.i18next.com/translation-function/plurals (for the docs)
This one isn't too bad, if you provide `t` with the parameter `count`, in addition to interpolating it into a translation file (if the variable exists), it also selects the plural interpretation of it

Reply:
so in this case, if `txs.length` is > 1 it'll select the <subkey>_plural instead of just <key>